### PR TITLE
Enable type checking

### PR DESCRIPTION
**Pull request by anandsaw**
### **Describe your change:**

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by cclauss** [comment]
Mypy is run here:
* https://github.com/TheAlgorithms/Python/blob/master/.github/workflows/pre-commit.yml
* https://github.com/TheAlgorithms/Python/blob/master/.pre-commit-config.yaml#L33

It is better to run such tools in pre-commit tools for users advanced enough to use pre-commit.

It would be great if you could uncomment those lines in `.pre-commit-config.yaml` and fix our files to be compatible with mypy.

**Comment by cclauss** [comment]
We are moving away from Travis --> GitHub Actions so it is better t add new automation there.

**Comment by anandsaw** [comment]
Whoops, sorry was trying to merge to my own fork and not to the main project. This is a test for a travis automation tool we are building.

File: test_mypy.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+python --version
+
+mypy_test() {
+
+    serverIp=""
+    serverPassword=""
+
+    pip install mypy
```
Comment (cclauss) [inline test_mypy.sh:10]: https://snarky.ca/why-you-should-use-python-m-pip/

**Pull request closed without merging**